test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/XPlatLocalsTests.cs

```diff
@@ -82,7 +82,8 @@ public static void Locals_Clear_Succeeds(string args)
                 // Arrange
                 var mockGlobalPackagesDirectory = Directory.CreateDirectory(Path.Combine(mockBaseDirectory.Path, @"global-packages"));
                 var mockHttpCacheDirectory = Directory.CreateDirectory(Path.Combine(mockBaseDirectory.Path, @"http-cache"));
-                var mockTmpCacheDirectory = Directory.CreateDirectory(Path.Combine(mockBaseDirectory.Path, @"temp"));
+                var mockTmpDirectory = Directory.CreateDirectory(Path.Combine(mockBaseDirectory.Path, @"temp"));
+                var mockTmpCacheDirectory = Directory.CreateDirectory(Path.Combine(mockBaseDirectory.Path, @"NuGetScratch"));
```

Comment:
mockTmpDirectory is a mock for c:/Users/..../Temp
mockTmpCacheDirectory is a mock for c:/Users/..../Temp/NuGetScratch

mockTmpCacheDirectory is contained within mockTmpDirectory, to match the folder structure as in actual Temp directory.